src/pineko/fonll.py

```diff
@@ -235,7 +239,7 @@ def collect_updates(fonll_fns, damp):
     """Produce the different theory cards according to which FONLL is asked for."""
     updates = []
     is_mixed = fonll_fns in MIXED_ORDER_FNS
-    is_damped = damp != 0
+    is_damped = damp >= 0
```

Comment:
done in [643313b](https://github.com/NNPDF/pineko/pull/142/commits/643313bc1707e4f3f6f6349b5a49bfaea49ecc23) - I also had to adjust the original fix since I broke the benchmarks .... (I guess I'm still confused by the fact that mixing and damping are the same ...)